changelog/util.go

```diff
@@ -0,0 +1,27 @@
+package changelog
```

Comment:
I could be missing something, but can this file and the one in `actions/changelog_util.go` be collapsed into one?  Or are both versions of each `commitFocus`/`commitTitle` necessary?